src/com/google/debugging/sourcemap/SourceMapGeneratorV3.java

```diff
@@ -137,7 +137,6 @@ public void reset() {
     mappings.clear();
     lastMapping = null;
     sourceFileMap.clear();
-    sourceFileContentMap.clear();
```

Comment:
This doesn't seem right.  It's very surprising that it would reset everything *except* this map.

I don't quite understand your assertion about being reset and added at the wrong time.  Once the source map is reset, all the mappings need to be re-added.  Would it be possible to simply ensure that, along with adding mappings, we also add the sources contents?

Reply:
Can the life cycle be changed to be more consistent?  Can we initialize `sourceFileContentMap` during source map generation?  Or pass the content map in as a separate argument later?  I'm just not comfortable having a `reset` method that doesn't actually reset everything.

Reply:
I just ran into this as well and need it fixed.

@WearyMonkey If you aren't comfortable making this change, I have an idea of how to approach it. It'll need handled in the GWT runner too.